From: SJ Park <sj@kernel.org>
Date: Tue, 4 Aug 2026 07:52:09 -0700
Subject: [PATCH] Revert "mm/damon/lru_sort: error out for >10000
 active_mem_bp"

This reverts commit dbc61ec2db641b2c20aeaa64ca238567b8adb8b7.

Now the core parameters validation handles the zero target_value case.
Remove unnecessary additional validation.

Signed-off-by: SJ Park <sj@kernel.org>
---
 mm/damon/lru_sort.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/mm/damon/lru_sort.c b/mm/damon/lru_sort.c
index f25ee7326e87c..ad8e86dd3a93e 100644
--- a/mm/damon/lru_sort.c
+++ b/mm/damon/lru_sort.c
@@ -233,8 +233,6 @@ static int damon_lru_sort_add_quota_goals(struct damos *hot_scheme,
 
 	if (!active_mem_bp)
 		return 0;
-	if (10000 < active_mem_bp)
-		return -EINVAL;
 	goal = damos_new_quota_goal(DAMOS_QUOTA_ACTIVE_MEM_BP, active_mem_bp);
 	if (!goal)
 		return -ENOMEM;
-- 
2.47.3

